celestia-app/private/config/client.toml

```diff
@@ -6,7 +6,7 @@
 ###############################################################################
 
 # The network chain ID
-chain-id = "private"
```

Comment:
This was called `private` because you couldn't override configs for networks that weren't called `private` in some version of node. Not sure if you've tested to make sure this works.